src/components/swap/Tabs.tsx

```diff
@@ -50,18 +52,33 @@ const StyledEcoRouter = styled(EcoRouter)`
   margin-right: 5px;
 `
 
-export const Tabs = () => {
+export function Tabs() {
```

Comment:
do we want to change functions to this style?

Reply:
Not really. Just a personal preference; I like to use regular functions when the body has some logic.